trac_configurable_ctu/__init__.py

```diff
@@ -93,6 +113,19 @@ def expand_macro(self, formatter, name, content, args=None):
         except Exception: # pylint: disable=broad-except
             message = content
             resource = Resource('repository', reponame)
+        config = self.ticket_replace_section
+        fields = {}
+        for key, value in config.options():
+            prefix, option = key.split('.')
+            field = fields.setdefault(prefix, {})
+            field[option] = config.get(key)
+        for prefix, field in fields.iteritems():
+            if not all (k in field for k in ['pattern', 'replace']):
+                continue
```

Comment:
Do you have a way to print a log message if this happens here, i.e. do macros have the `self.log` member? Silently ignoring things is always hard to debug.